R/sf.R

```diff
@@ -511,6 +507,21 @@ CoordSf <- ggproto("CoordSf", CoordCartesian,
     graticule$y_start <- sf_rescale01_x(graticule$y_start, y_range)
     graticule$y_end <- sf_rescale01_x(graticule$y_end, y_range)
 
+    # Convert 'degree' to the degree symbol
+    parse_ids <- grepl("\\bdegree\\b", graticule$degree_label)
+    if (any(parse_ids)) {
+      graticule$degree_label <- Map(
+        function(parse_id, label) {
+          if (parse_id) {
+            parse(text = label)[[1]]
+          } else {
+            as.expression(label)[[1]]
+          }
+        },
+        parse_ids, graticule$degree_label
+      )
+    }
+
```

Comment:
Would you mind moving this code block into the `fixup_graticule_labels()` function?